external/runtime/runtime.depproj

```diff
@@ -133,5 +134,14 @@
       <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPathsNativeImages)" />
     </ItemGroup>
 
+    <!-- For coverage runs of any crossgen dll from CoreClr we need the respective IL, the pdbs are already in place -->
+    <Copy SourceFiles="@(CoreCLRILFiles)"
+          DestinationFolder="$(NETCoreAppTestSharedFrameworkPath)/il"
+          SkipUnchangedFiles="true" />
+    <!-- Make backups of the crossgen dlls that do not have .ni. in their names, so they can be restored after coverage runs  -->
```

Comment:
Something is automatically restoring them?  What happens if I ctrl-C during a coverage run... will that cause subsequent problems?

Reply:
That is not addressed on this change, this only copy the files to the location under testhost/**/9.9.9 so we can restore them, the idea being that having a copy at hand under ni allows us to check on the start and if it is not coverage ensure that the native image is in place. Notice also that for now this doesn't cover the package but only when using CoreClrOverridePath.

Reply:
One thing that I need to be careful in this regard is that by default, using CoreClrOverridePath or not, CoreLib is a hardlink. This can be messy and confuse, perhaps the path here is to break the link on coverage runs and restore the link when running tests again.